flathub.json

```diff
@@ -0,0 +1,3 @@
+{
```

Comment:
you can just remove this file. `aarch64` and `x86_64` are the only two arch supported anyway.